SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -58,21 +60,22 @@ FNetworkGUID USpatialPackageMapClient::ResolveEntityActor(AActor* Actor, FEntity
 	return NetGUID;
 }
 
-void USpatialPackageMapClient::RemoveEntityActor(const FEntityId& EntityId)
+void USpatialPackageMapClient::RemoveEntityActor(Worker_EntityId EntityId)
 {
 	FSpatialNetGUIDCache* SpatialGuidCache = static_cast<FSpatialNetGUIDCache*>(GuidCache.Get());
-	if (SpatialGuidCache->GetNetGUIDFromEntityId(EntityId.ToSpatialEntityId()).IsValid())
+
+	if (SpatialGuidCache->GetNetGUIDFromEntityId(EntityId).IsValid())
 	{
-		SpatialGuidCache->RemoveEntityNetGUID(EntityId.ToSpatialEntityId());
+		SpatialGuidCache->RemoveEntityNetGUID(EntityId);
 	}
 }
 
-void USpatialPackageMapClient::RemoveEntitySubobjects(const FEntityId& EntityId, const SubobjectToOffsetMap& SubobjectToOffset)
+void USpatialPackageMapClient::RemoveEntitySubobjects(Worker_EntityId EntityId)
```

Comment:
Now that we don't remove this on the metadata change, we should just perform this functionality in the `RemoveEntityActor` function.